content/en/docs/concepts/workloads/pods/pod-topology-spread-constraints.md

```diff
@@ -212,18 +212,48 @@ profiles:
       args:
         defaultConstraints:
           - maxSkew: 1
-            topologyKey: failure-domain.beta.kubernetes.io/zone
+            topologyKey: topology.kubernetes.io/zone
             whenUnsatisfiable: ScheduleAnyway
 ```
 
 {{< note >}}
 The score produced by default scheduling constraints might conflict with the
 score produced by the
-[`DefaultPodTopologySpread` plugin](/docs/reference/scheduling/config/#scheduling-plugins).
+[`SelectorSpread` plugin](/docs/reference/scheduling/config/#scheduling-plugins).
 It is recommended that you disable this plugin in the scheduling profile when
 using default constraints for `PodTopologySpread`.
 {{< /note >}}
 
+#### Internal default constraints
+
+{{< feature-state for_k8s_version="v1.19" state="alpha" >}}
```

Comment:
Can you update the [Feature gate page](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/) as well? You need to do the following:

- Add the `DefaultPodTopologySpread` feature gate to the feature gate table
- Add a short description of `DefaultPodTopologySpread` to the list of feature gates (and link back to this page of docs!)

Reply:
Done